docs/dev/running/aziot-identityd.md

```diff
@@ -12,10 +12,6 @@ homedir = "/var/lib/aziot/identityd"
 
 [provisioning]
 
-[endpoints]
```

Comment:
Every service documents them because these are dev docs. So if you're going to remove this here you need to remove it from all the others as well.

But these are dev docs, and the expected reader is us devs. I suggest restoring it for now, with the entry added for tpmd to fix that GH issue. Then we need to figure out a way so that non-devs don't find these docs under the dev/ directory. Hide them from the default sidebar unless you're under dev/ or something, perhaps.

Reply:
Fair enough. In that case, I'll just re-add `[endpoints]` to the example config + add a `tpmd` entry as well.